README.md

```diff
@@ -158,6 +159,32 @@ $webhook = $pusher->webhook($request_headers, $request_body);
 $number_of_events = count($webhook->get_webhooks());
 ```
 
+## End to end encryption (beta)
```

Comment:
I think this could still benefit from https://github.com/pusher/pusher-http-php/pull/171#discussion_r196421528

Reply:
I can't see which comment you're referring to :(

Reply:
> This should mention the requirement to
> 
> * install the libsodium-dev system package (and how to do this on various platforms)
> * use either PHP 7.2 or installing the correct pecl extension for the version of PHP they are using
> * enable the config in sodium.ini